packages/core/helper-plugin/lib/src/hooks/useTracking/index.js

```diff
@@ -5,22 +5,31 @@ import useAppInfos from '../useAppInfos';
 
 const useTracking = () => {
   const trackRef = useRef();
-  const { uuid, telemetryProperties } = useContext(TrackingContext);
+  const { uuid, telemetryProperties, deviceId } = useContext(TrackingContext);
   const appInfo = useAppInfos();
+  const adminUserId = appInfo?.adminUserId;
 
   trackRef.current = async (event, properties) => {
     if (uuid && !window.strapi.telemetryDisabled) {
       try {
-        await axios.post('https://analytics.strapi.io/track', {
-          event,
-          properties: {
-            ...telemetryProperties,
-            ...properties,
-            projectType: window.strapi.projectType,
-            environment: appInfo.currentEnvironment,
+        await axios.post(
+          'https://analytics.strapi.io/api/v2/track',
+          {
+            event,
+            adminUserId,
```

Comment:
Should this just be called userId ?

Reply:
@fdel-car was also advocating for this. Changed it 👍